Library/Homebrew/extend/os/linux/install.rb

```diff
@@ -17,9 +17,28 @@ module Install
     ].freeze
     private_constant :DYNAMIC_LINKERS
 
+    GCC_VERSION_SUFFIX = OS::LINUX_GCC_CI_VERSION.sub("\.0", "").freeze
```

Comment:
```suggestion
    GCC_VERSION_SUFFIX = OS::LINUX_GCC_CI_VERSION.sub("\.0$", "").freeze
```

maybe?

Reply:
I'll test that this works but it's probably the right change to make.

Reply:
If you make it a `Regexp` (`/\.0$/`) it will remove the `.0`.

Reply:
Ah that was the issue.  I'll test to double check but this makes sense as a solution.

Reply:
This `Regexp` works and I've pushed the change.